ckanext/who_afro/templates/group/read.html

```diff
@@ -1,14 +1,91 @@
+    {% set placeholder = placeholder if placeholder else _('Search datasets...') %}
+    {% set sorting = sorting if sorting else [(_('Name Ascending'), 'name asc'), (_('Name Descending'), 'name desc')] %}
+    {% set search_class = search_class if search_class else 'search-giant' %}
+    {% set no_bottom_border = no_bottom_border if no_bottom_border else false %}
+    {% set form_id = form_id if form_id else false %}
+
+    <h2> {{ _('Search data') }}</h2>
+
+    <form {% if form_id %}id="{{ form_id }}" {% endif %}class="search-form{% if no_bottom_border %} no-bottom-border{% endif %}"  method="get" action="{% url_for 'dataset.search' %}">
+        <div class="input-group">
+          <input aria-label="{% block header_site_search_label %}{{ placeholder }}{% endblock %}" id="field-giant-search" type="text" class="form-control input-lg" name="q" value="{{ query }}" autocomplete="off" placeholder="{{ placeholder }}">
+
+          <span class="input-group-btn">
+            <button class="btn btn-default btn-lg lens-btn" type="submit" value="search" aria-label="{{_('Submit')}}">
+                <i class="fa fa-search"></i><span class="search-label">{{ _('Search') }}</span>
+            </button>
+          </span>
+        </div>
+    </form>
+  </div>
+{% endblock %}
+
+
 {% block secondary_content %}
-  {% snippet "group/snippets/info.html", group=group_dict, show_nums=true %}
+  {% set sorting = [(_('Relevance'), 'score desc, metadata_modified desc'), (_('Name Ascending'), 'name asc'), (_('Name Descending'), 'name desc')] %}
+  {% set sorting_selected = request.args.get('sort') %}
   <div class="filters">
     <div>
-      {% for facet in facet_titles %}
-        {% set scheming_choices=h.scheming_field_by_name(h.scheming_get_dataset_schema(dataset_type).dataset_fields, facet).choices %}
-        {{ h.snippet('snippets/facet_list.html', title=facet_titles[facet], name=facet, search_facets=search_facets, scheming_choices=scheming_choices) }}
-      {% endfor %}
+      <h3>{{ _('Order by:') }}</h3>
+      <div class="form-group control-order-by">
+        <!-- <label for="field-order-by">{{ _('Order by') }}</label> -->
+        <select id="field-order-by" name="sort" class="form-control form-select" onchange="setSort(this.value);">
+          {% for label, value in sorting %}
+            {% if label and value %}
+              <option value="{{ value }}"{% if sorting_selected == value %} selected="selected"{% endif %}>{{ label }}</option>
+            {% endif %}
+          {% endfor %}
+        </select>
+        {% block search_sortby_button %}
+        <button class="btn btn-default js-hide" type="submit">{{ _('Go') }}</button>
+        {% endblock %}
+      </div>
+
+      <h3>{{ _('Filter by:') }}</h3>
+      <div class="accordion" id="filters">
+        {% for facet in facet_titles %}
+          {% set scheming_choices=h.scheming_field_by_name(h.scheming_get_dataset_schema(dataset_type).dataset_fields, facet).choices %}
```

Comment:
```suggestion
          {% if facet in ["programme", "country"] %}
            {% set scheming_choices=h.scheming_field_by_name(h.scheming_get_dataset_schema(dataset_type).dataset_fields, facet).choices %}
            {{ h.snippet('snippets/facet_list.html', title=facet_titles[facet], name=facet, search_facets=search_facets, scheming_choices=scheming_choices) }}
          {% endif %}
```

Reply:
@ChasNelson1990  I just tried pushing my changes of yesterday following the recommendation by @jonathansberry and you most certainly have the other solution we were considering. Will sync with @jonathansberry about which one to keep.

Reply:
I don't think you quite understood me Ntwali.  And I'm not sure Chas' approach is ideal either, as it changes it for all groups. My suggestion was to get the same result as chas, but to do it using CSS. I've opened a little PR showing what I would do: https://github.com/fjelltopp/ckanext-who-afro/pull/61